frontend/src/components/EnvironmentForm/AccessControlConfigure.vue

```diff
@@ -61,7 +61,12 @@
       </div>
     </div>
   </div>
-  <div v-if="databaseChangeMode === 'PIPELINE'" class="flex flex-col gap-y-2">
+  <div
+    v-if="
+      databaseChangeMode === 'PIPELINE' && resource.startsWith('environments/')
```

Comment:
It's better to use `environmentNamePrefix` https://github.com/bytebase/bytebase/blob/main/frontend/src/store/modules/v1/common.ts#L7